docs: do not use Palette Edge CLI for upgrade to 4.6.16 or later (#6729)
spectrocloud/librarium: docs/docs-content/clusters/edge/cluster-management/agent-upgrade-airgap.md

   [Export Cluster Definition](../local-ui/cluster-management/export-cluster-definition.md) to build a content bundle
   using your new cluster profile and export the cluster definition.

   :::warning

>    If you are upgrading to an agent version that is 4.6.16 or later, use the Palette CLI. Do not use the Palette Edge

Reviewer: ```suggestion
   If you're upgrading to an agent version 4.6.16 or later, use the Palette CLI instead of the Palette Edge CLI.
```

>    If you are upgrading to an agent version that is 4.6.16 or later, use the Palette CLI. Do not use the Palette Edge

Reviewer: ```suggestion
   If you are upgrading to an agent version 4.6.16 or later, use the Palette CLI instead of the Palette Edge
```

Author: I am not sure about this one. The Palette CLI is already the recommended approach. I want to emphasize that they must not use the Edge CLI so I wanted to use two sentences.
